cmake/Tools/Platform/Android/unit_test_android_deployment.py

```diff
@@ -658,14 +658,14 @@ def _mock_adb_call(arg_list, device_id=None):
 
 @pytest.mark.parametrize(
     "test_game_name, test_config, test_package_name, test_device_storage_path, test_asset_type", [
-        pytest.param('game1','profile', 'com.amazon.lumberyard.foo', '/data/fool_storage', 'android'),
-        pytest.param('game1','debug', 'com.amazon.lumberyard.foo', '/data/fool_storage', 'android'),
-        pytest.param('game2','profile', 'com.amazon.lumberyard.bar', '/data/fool_storage', 'android'),
-        pytest.param('game2','debug', 'com.amazon.lumberyard.bar', '/data/fool_storage', 'android'),
-        pytest.param('game3','profile', 'com.amazon.lumberyard.foo', '/data/fool_storage2', 'pc'),
-        pytest.param('game3','debug', 'com.amazon.lumberyard.foo', '/data/fool_storage2', 'pc'),
-        pytest.param('game4','profile', 'com.amazon.lumberyard.bar', '/data/fool_storage2', 'pc'),
-        pytest.param('game4','debug', 'com.amazon.lumberyard.bar', '/data/fool_storage2', 'pc')
+        pytest.param('game1','profile', 'com.amazon.o3de.foo', '/data/fool_storage', 'android'),
+        pytest.param('game1','debug', 'com.amazon.o3de.foo', '/data/fool_storage', 'android'),
+        pytest.param('game2','profile', 'com.amazon.o3de.bar', '/data/fool_storage', 'android'),
+        pytest.param('game2','debug', 'com.amazon.o3de.bar', '/data/fool_storage', 'android'),
+        pytest.param('game3','profile', 'com.amazon.o3de.foo', '/data/fool_storage2', 'pc'),
+        pytest.param('game3','debug', 'com.amazon.o3de.foo', '/data/fool_storage2', 'pc'),
+        pytest.param('game4','profile', 'com.amazon.o3de.bar', '/data/fool_storage2', 'pc'),
+        pytest.param('game4','debug', 'com.amazon.o3de.bar', '/data/fool_storage2', 'pc')
```

Comment:
Same here regarding the ".amazon" portion

Reply:
Done.